ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -107,10 +108,18 @@ private struct MainSection: View {
                     label: "settings_licenses_title",
                     action: { activeSection = .licenses }
                 )
+                if !RuntimeInfo.isProduction {
```

Comment:
I actually think we could just _always_ show this, hm? Particularly considering that it doesn't actually send anything to us unless the user explicitly does so... We could hide it in a _Developer_ section or something maybe?